client.go

```diff
@@ -118,6 +126,28 @@ func NewTokenClient(key string, userAgent string) (*Client, error) {
 	return c, nil
 }
 
+// SetupClient creates a new client using the provided web URL and token in the environment.
+// This should only be used for testing and development purposes, or if you know what you're doing.
+func SetupClient(userAgent string) (*Client, error) {
```

Comment:
Since we only use it for testing or dev purpose and testing, can we move it to a separate file and use build tag? 
```
//go:build dev
```

Reply:
Also, I think the name is lacking context. Would you consider renaming it something like `SetupClientFromTheEnvVars`?

Reply:
> Since we only use it for testing or dev purpose and testing, can we move it to a separate file and use build tag?

I'll consider it in a separate PR

Reply:
> Also, I think the name is lacking context. Would you consider renaming it something like `SetupClientFromTheEnvVars`?

I've renamed it to `SetupEnvClient`, however I think it's absolutely justified to have it here since it could be a useful function outside of dev-only scope, the part of the comment says that it could be used if:

> ...you know what you're doing.

thus I propose to keep it as is